### PR TITLE
Enhancement/new project modal

### PR DESCRIPTION
**Pull request by jeanawhou**
Ticket: https://airtable.com/appfbr7vlSqIfZ9Sn/tblP6AWIpvkOYZ0jR/viwou8K2UyUSkWGts/rec0jimoZ7PGjrFDu?blocks=hide

@Quiescent @cameronsutter i was able to create new project with the use cases listed below 😅 

Use cases
On Main Dashboard (blank project)
1. Click `Create Blank Project` panel button ✅ 
2. Click `File` Menu > `Create Blank Project` ✅

On Main Dashboard (with template)
1. Click `Create From Template` panel button ✅
2.  Click `File` Menu > `Create From Template` ✅
 
In an open project (blank project)
1. Click Nav `Dashboard` button > on Dashboard Modal > click `Create Blank Project` panel button ✅
2. Click `File` Menu > `Create Blank Project` ✅

In an open project (with template)
1. Click Nav `Dashboard` button > on Dashboard Modal > click `Create From Template` panel button ✅
2. Click `File` Menu > `Create From Template` ✅

### CODE REVIEW

File: lib/plottr_components/src/components/dashboard/files/FilesHome.js [hunk 1]
```diff
@@ -62,20 +63,8 @@ const FilesHomeConnector = (connector) => {
     }
   }
 
-  const FilesHome = ({ errorActions, importActions, isOnWeb }) => {
+  const FilesHome = ({ errorActions, importActions, isOnWeb, projectActions }) => {
     const [view, setView] = useState('recent')
-    const [namingFile, setNamingFile] = useState(null)
```
Comment (Quiescent) [inline lib/plottr_components/src/components/dashboard/files/FilesHome.js:67]: + 1 million code points for removing this state and simplifying the component.
Great job! 💯

**Comment by Quiescent** [comment]
@jeanawhou, can we create a new project when the timeline is visible and the dashboard modal is closed?

File: src/app/index.js [hunk 2]
```diff
@@ -353,7 +353,9 @@ const reloadMenu = () => ipcRenderer.send('pls-reload-menu')
 window.addEventListener('load', reloadMenu)
 window.addEventListener('focus', reloadMenu)
 
-ipcRenderer.on('new-project', () => createBlankProj())
+ipcRenderer.on('new-project', () => {
+  store.dispatch(actions.project.startCreatingNewProject(true))
```
Comment (Quiescent) [inline src/app/index.js]: @jeanawhou, I think you missed this one.  Wouldn't this set the template to `true`?

**Comment by Quiescent** [comment]
@jeanawhou, this is excellent work!
I'm thrilled with the quality of this change; it both fixes a bug and improves all of the components it touches!
💯 👍 🙌 

(@cameronsutter FYI.)

**Comment by jeanawhou** [comment]
> @jeanawhou, can we create a new project when the timeline is visible and the dashboard modal is closed?

Yes @Quiescent it should be able to create new project without the dashboard/dashboard modal 😅

**Review by Quiescent — APPROVED** [review]
LGTM!

File: src/app/index.js (hunk 2 above)
Comment (jeanawhou) [reply src/app/index.js]: @Quiescent thank you for checking 🙇‍♀️

**Comment by jeanawhou** [comment]
> @jeanawhou, this is excellent work! I'm thrilled with the quality of this change; it both fixes a bug and improves all of the components it touches! 💯 👍 🙌
> 
> (@cameronsutter FYI.)

Thank you so much it means a lot to me 🥲

**Comment by Quiescent** [comment]
> > @jeanawhou, this is excellent work! I'm thrilled with the quality of this change; it both fixes a bug and improves all of the components it touches! 💯 👍 🙌
> > (@cameronsutter FYI.)
> 
> Thank you so much it means a lot to me 🥲

My pleasure 🙂

**Pull request merged**